### PR TITLE
Support for output streams other than the default console

### PR DESCRIPTION
**Pull request by gonenduk**
Allows the output text in verbose mode to be sent to log files instead of the console.

### CODE REVIEW

**Comment by gonenduk** [comment]
bump

File: src/index.coffee [hunk 1]
```diff
@@ -18,9 +18,16 @@ master = (config) ->
       true
     else
       Boolean config.respawn
+
+  outputStream =
+    if config.outputStream and typeof config.outputStream.write is "function"
+      config.outputStream.write
```
Comment (eden) [inline src/index.coffee]: If I recall correctly, `WritableStream.write` doesn't add a newline to the outputted string, so this doesn't behave similarly to `console.log`. I'd suggest making `outputStream` a function and adding the newline manually if its given in the config.

**Review by eden — COMMENTED** [review]
Looks great. Thanks for the contribution. Just one small change requested.

I don't have write access to this repo anymore, but I'm sure someone can merge this change in when they get a chance to look at it.

**Comment by gonenduk** [comment]
Hey Eden, thanks for the review.

1. Do you know who has write permissions and can merge the pull request?
2. The idea behind this change is to send the output to a log file. I use this to write to the Winston logger. Just like console.log, in loggers there is no need to add a new line at the end. If I add a new line it will mess the log.

**Comment by eden** [comment]
> 1. Do you know who has write permissions and can merge the pull request?

I think @dcreemer will know best.

> 2. The idea behind this change is to send the output to a log file. I use this to write to the Winston logger. Just like console.log, in loggers there is no need to add a new line at the end. If I add a new line it will mess the log.

Interesting. I'm not familiar with winston, but I looked into it a little but didn't see any way to easily pull out a stream that does writing this way. It seems that it can mimic `console.log` but only using the specific methods on its `Logger` class (e.g. `log`, `info`, `warn`, etc). Am I missing a common use case? Maybe it'll be instructive to see an example value that you'd pass to the `outputStream` key.

**Comment by gonenduk** [comment]
that's the code I use to initialize the logger (very standard):
`
const winston = require('winston');
const logger = new (winston.Logger)();
logger.stream = {
  write: (message, encoding) => {
    logger.info(message.trim());
  };
`

And that's the code used to set express-cluster with my change to use the logger instead of the console:
`
const cluster = require('express-cluster');
config.cluster.outputStream = logger.stream;
cluster(config.cluster, worker => {
});
`

**Comment by gonenduk** [comment]
bump

**Comment by gonenduk** [comment]
bump

**Comment by dcreemer** [comment]
sorry for missing this @gonenduk -- I'll get it committed

**Comment by gonenduk** [comment]
Please update version and publish to npm

**Comment by dcreemer** [comment]
I've bumped the version to 0.0.5. @eden can you push to npm?

**Comment by eden** [comment]
Published!

**Pull request merged**